src/NServiceBus.RabbitMQ/Connection/ConnectionFactory.cs

```diff
@@ -55,7 +56,6 @@ public IConnection CreateConnection(string connectionName)
         {
             lock (lockObject)
             {
-                connectionFactory.ClientProperties["purpose"] = connectionName;
```

Comment:
@bording I presume this is also appropriate in the 4.0 transport after upgrading the client to 3.6.3?